src/CMakeLists.txt

```diff
@@ -9,6 +8,66 @@ include_directories("md/inc")
 include_directories("classlibnative/bcltype")
 include_directories("classlibnative/cryptography")
 include_directories("classlibnative/inc")
+include_directories("${GENERATED_INCLUDE_DIR}")
+
+set (ScriptGeneratedEventFiles
+      ${GENERATED_INCLUDE_DIR}/clretwallmain.h
+      ${GENERATED_INCLUDE_DIR}/etmdummy.h
+    )
+set (GeneratedEventFiles)
+
+if(WIN32)
```

Comment:
I am sorry, please disregard this comment. I have missed the fact that the generated clretwallmain.h includes the generated ClrEtwAll.h.